### PR TITLE
[DJADTECH-3931] - Upgrade GMA SDK Version for all iOS Apps

### PR DESCRIPTION
**Pull request by BoogieMAN2K**
Google has send reminder to upgrade to the newest GMA SDK versions: 10 (iOS) and 22 (Android). 

At present we are not on the latest version of iOS at this time. 

Attaching the deprecation schedule for both [iOS](https://urldefense.com/v3/__https://developers.google.com/ad-manager/mobile-ads-sdk/ios/deprecation__;!!F0Stn7g!DsNbvFnflthAgajiDsjBeLqi9G8hL6GReqKlgjdjavbviXYuO6OnxN5OkSV3O19RfbDVeHXhwixNI_LmpLu7Cg2SuQ$) and [Android](https://urldefense.com/v3/__https://developers.google.com/ad-manager/mobile-ads-sdk/android/deprecation__;!!F0Stn7g!DsNbvFnflthAgajiDsjBeLqi9G8hL6GReqKlgjdjavbviXYuO6OnxN5OkSV3O19RfbDVeHXhwixNI_LmpLuT3OTcbw$), which includes migration guides for upgrading to a current version.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -16,27 +16,27 @@ let package = Package(
...
+            path: "GoogleMobileAds/10.12.0/GoogleAppMeasurement.xcframework"
         ),
         .binaryTarget(
             name: "GoogleUtilities",
-            path: "GoogleMobileAds/7.69.0/GoogleUtilities.xcframework"
+            path: "GoogleMobileAds/10.12.0/GoogleUtilities.xcframework"
         ),
         .binaryTarget(
             name: "PromisesObjC",
             path: "GoogleMobileAds/7.69.0/PromisesObjC.xcframework"
```
Comment (3257) [inline Package.swift:31]: Aren't `PromisesObjC` getting updated too?

**Comment by 3257** [comment]
These binaries are needed for other projects that use this dependency, is that correct?

File: Package.swift (hunk 1 above)
Comment (BoogieMAN2K) [reply Package.swift:31]: The zip that comes from Google doesn't have that library so I didn't update that one.
Screenshot 2023-10-27 at 14 50 53

**Comment by BoogieMAN2K** [comment]
> These binaries are needed for other projects that use this dependency, is that correct?

Correct.

**Review by 3257 — APPROVED** [review]

**Review by bhaskar-dj — APPROVED** [review]

**Pull request merged**